src/test/k6/README.md

```diff
@@ -0,0 +1,26 @@
+# Web Browser Performance Testing with K6
+
+Here we have written performance tests for the three services in Family Hubs: Find, Connect, Manage.
+# Prerequisites
+
+- Install k6 locally
+- Run data seed scripts
+- Navigate to folder of service under test (i.e. find-tests, connect-tests, or manage-tests)
+
+# How to run the tests
+Set the environment variables in settings.json to define whether you are running the tests locally and which type of test you are running (load, stress, or soak)
```

Comment:
I'd suggest writing some short one-liners (/ one-sentencers) on what each one of the test types means / does (load, stress, soak)